src/locales/id/duration.js

```diff
@@ -0,0 +1,2 @@
+import { id as locale } from '@musement/iso-duration';
```

Comment:
id doesn't exist in this package afaik.

We need to translate the following:

```suggestion
const lang: LangConfig = {
  years(c) {
    return "year" + (c === 1 ? "" : "s");
  },
  months(c) {
    return "month" + (c === 1 ? "" : "s");
  },
  weeks(c) {
    return "week" + (c === 1 ? "" : "s");
  },
  days(c) {
    return "day" + (c === 1 ? "" : "s");
  },
  hours(c) {
    return "hour" + (c === 1 ? "" : "s");
  },
  minutes(c) {
    return "minute" + (c === 1 ? "" : "s");
  },
  seconds(c) {
    return "second" + (c === 1 ? "" : "s");
  },
  decimal: ".",
};
```

and then we probably want to contribute it back to https://github.com/musement/iso-duration